submissions/pozzitive11/popup/index.html

```diff
@@ -29,97 +29,133 @@
+                    <p class="popup__link-descr">Apps</p>
+                  </a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link">
+                    <img class="popup__link-img" src="./images/documents.png" alt="document">
+                    <p class="popup__link-descr">Document</p>
+                  </a>
+                </li>
+                <input id="popup-extra" class="popup__extra-input" type="checkbox">
+                <label class="nav__btn popup__extra-btn" for="popup-extra">More</label>
+                <ul class="list-reset popup__list-extra">
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img class="popup__link-img" src="./images/chat.png" alt="chat">
+                      <p class="popup__link-descr">Chat</p>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img class="popup__link-img" src="./images/classroom.png" alt="classroom">
+                      <p class="popup__link-descr">Classroom</p>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img class="popup__link-img" src="./images/videos.png" alt="collection">
+                      <p class="popup__link-descr">Videos</p>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img class="popup__link-img" src="./images/mars.png" alt="mars">
+                      <p class="popup__link-descr">Mars</p>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img class="popup__link-img" src="./images/blogs.png" alt="blogs">
+                      <p class="popup__link-descr">Blogs</p>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link">
+                      <img class="popup__link-img" src="./images/promo.png" alt="promos">
+                      <p class="popup__link-descr">Promos</p>
+                    </a>
+                  </li>
                   <a href="https://github.com/kottans/frontend" class="menu__extra-link">
                     Even more from Kottans
                   </a>
-                </div>
-              </div>
+                </ul>
+              </ul>
             </div>
           </li>
           <li class="nav__item">
             <a href="#" class="nav__btn">
-              <img class="avatar" src="images/bell.png" alt="Notification icon">
+              <img class="avatar" src="images/bell.png" alt="">
```

Comment:
You set `alt` attribute with an empty value for this icon, what about other icons, or do you have only one icon in your popup?)